install.sh

```diff
@@ -524,6 +530,59 @@ __package_installed() {
 	fi
 }
 
+__package_version() {
+	if ! __package_installed "$1"; then
+		echo ""
+	fi
+
+	if [ $_PKG_MGR -eq 1 ]; then # apt
+		echo $(dpkg-query -W -f '${Version}' "$1")
+	elif [ $_PKG_MGR -eq 2 ]; then # yum and dnf
+		echo $(rpm -qa --queryformat='%{VERSION}' "$1")
+	fi
+}
+
+__satisfies_version() {
```

Comment:
`__satisfies_version` seems like a prime candidate for unit testing. Does anyone have ideas on how to actually import this to use in a unit test? Running `source install.sh` in another file will cause issues because there are lines that aren't in functions that are run (including the entrypoint). If we split functions like this out into another script file we lose the benefit of having a user download one installer file in order to install RITA.